modules/migrating-sccs-during-the-manual-upgrade.adoc

```diff
@@ -0,0 +1,360 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrade-roxctl.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="migrating-sccs-during-the-manual-upgrade_{context}"]
+= Migrating SCCs during the manual upgrade
+
+By migrating the security context constraints (SCCs) during the manual upgrade by using `roxctl` CLI, you can seamlessly transition the {rh-rhacs-first} services to use the {osp} SCCs, ensuring compatibility and optimal security configurations across Central and all secured clusters.
+
+.Procedure
+
+. List all of the {product-title-short} services that are deployed on Central and all secured clusters:
++
+[source,terminal]
+----
+$ oc -n stackrox describe pods | grep 'openshift.io/scc\|^Name:'
+----
++
+.Example output
++
+[source,text,subs="+quotes"]
+----
+Name:      admission-control-6f4dcc6b4c-2phwd
+           *openshift.io/scc: stackrox-admission-control*
+#...
+Name:      central-575487bfcb-sjdx8
+           *openshift.io/scc: stackrox-central*
+Name:      central-db-7c7885bb-6bgbd
+           *openshift.io/scc: stackrox-central-db*
+Name:      collector-56nkr
+           *openshift.io/scc: stackrox-collector*
+#...
+Name:      scanner-68fc55b599-f2wm6
+           *openshift.io/scc: stackrox-scanner*
+Name:      scanner-68fc55b599-fztlh
+#...
+Name:      sensor-84545f86b7-xgdwf
+           *openshift.io/scc: stackrox-sensor*
+#...
+----
++
+In this example, you can see that each pod has its own custom SCC, which is specified through the `openshift.io/scc` field.
+
+. Add the required roles and role bindings to use the {osp} SCCs instead of the {product-title-short} custom SCCs.
```

Comment:
This technically includes the following 2 steps, so I would put it together with the next step. As it is now, it's a step with no actual content in it.

Reply:
So just combine:

. Add the required roles and role bindings to use the {osp} SCCs instead of the {product-title-short} custom SCCs. To add the required roles and role bindings to use the {osp} SCCs for the Central cluster, perform the following steps: